### DISCUSSION TITLE
Does GitHub have a chat feature?

### DISCUSSION THREAD

**Discussion by jupiterwebdev** [community/community#37018] [Other Feature Feedback, Questions, & Ideas] [answered]
Does GitHub have a chat feature?

**Comment by ra0321** [comment, marked as answer]
Does GitHub have a chat feature?
Set up the GitHub app in Chat

and select the GitHub repository to get notifications for. Under Thread messages by, select No threads or Repository.

**Reply by Talalgmel** [reply]
Y
